uint8 sigma_x
uint8 sigma_y
uint8 sigma_z
uint8 sigma_roll
uint8 sigma_pitch
uint8 sigma_yaw
